packages/theme/composables/useMakeOrder/commands/placeOrderCommand.ts

```diff
@@ -1,9 +1,10 @@
-import { PlaceOrderOutput } from '~/modules/GraphQL/types';
+import type { NuxtAppOptions } from '@nuxt/types';
+import type { PlaceOrderOutput } from '~/modules/GraphQL/types';
 
 export const placeOrderCommand = {
-  execute: async (context, cartId: string): Promise<PlaceOrderOutput> => {
-    const { data } = await context.$vsf.$magento.api.placeOrder({ cart_id: cartId });
+  execute: async (app: NuxtAppOptions, cartId: string): Promise<PlaceOrderOutput | null> => {
```

Comment:
```suggestion
  execute: async (context: Context, cartId: string): Promise<PlaceOrderOutput | null> => {
```

All our other *Command.ts files use context as the arg here, so even if the @nuxt/types Context doesn't work, the not-so-beautiful Return<useContext> type is better than changing the convention